### PR TITLE
Styling updates

### PR DESCRIPTION
**Pull request by breville**
Some general styling updates have been made.

Also, in particular:

- The statement has been restored to DataDisplay and is also shown in TrainModel, Results, and SaveModel.
- SelectDataset now shows mouse hover highlighting.
- DataDisplay now styles its columns for highlight (due to mouse hover) & selected, and column headers for label & feature.
- The PhraseBuilder now uses the terms label & feature.
- TrainingSettings is now hidden if both `hideSpecifyColumns` & `hideChooseReserve` parameters are set.
- Going back from Results goes to feature selection, while going back from feature selection clears features and goes to label selection, making for a symmetrical forward & back navigation.
- The prediction & actual columns have been swapped so that actual results are closest to the source data.
- The predict button now sizes for its content & goes to half-opacity when disabled.
- Arrows are removed from the navigation buttons.
- The buttons no longer slightly overlap the UI in the standalone version.

### screenshots

Screen Shot 2021-03-31 at 8 46 10 AM

Screen Shot 2021-03-31 at 8 44 00 AM

Screen Shot 2021-03-31 at 8 44 19 AM

Screen Shot 2021-03-31 at 8 44 33 AM

Screen Shot 2021-03-31 at 8 44 50 AM

Screen Shot 2021-03-31 at 8 45 02 AM

### CODE REVIEW

File: src/UIComponents/Statement.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/* React component to display a statement about our model. */
```
Comment (Erin007) [inline src/UIComponents/Statement.jsx:1]: Hooray for extracted components!

File: src/constants.js [hunk 2]
```diff
@@ -225,60 +238,65 @@ export const styles = {
...
+  dataDisplayHeader: {
+    paddingLeft: 20,
+    textAlign: "right",
+    position: "sticky",
+    top: 0,
+    backgroundColor: "white",
+    color: "#4d575f",
+    borderStyle: "solid",
+    borderWidth: 1,
+    borderColor: "white", // #c6cacd",
```
Comment (Erin007) [inline src/constants.js:250]: tiny nit: Do we need these little comments here and line 171? I'm not sure when we're using words, when hex and when rgb and why.

**Comment by Erin007** [comment]
Screen Shot 2021-03-30 at 4 30 43 PM
While you're doing a style pass is it worth making "Features" the same size as "Actual" and "A.I. Prediction" and left aligning it?

**Comment by Erin007** [comment]
Screen Shot 2021-03-30 at 4 33 07 PM

Is partial census data also selected or previously selected? Not sure why it also looks like it has a blue outline.

**Review by Erin007 — APPROVED** [review]
A few little things to consider, but nothing ship blocking.

File: src/constants.js (hunk 2 above)
Comment (breville) [reply src/constants.js:250]: Oh, I will remove this detritus.

**Comment by breville** [comment]
> While you're doing a style pass is it worth making "Features" the same size as "Actual" and "A.I. Prediction" and left aligning it?

Funnily, I thought it might have been deliberate for some reason.  Will fix.

**Review by made-line — APPROVED** [review]
Looks good! I like the symmetrical forward and back navigation. 

Do we want to keep the different styling for the data table (gray background) and results table (white background)?

**Comment by breville** [comment]
> Do we want to keep the different styling for the data table (gray background) and results table (white background)?

I was wondering that too.  I think we might do some more work on Results soon, and can consider that then.  I think it might be worth differentiating them, though not entirely clear how, yet.

**Pull request merged**